core/platform/auth/firebase_auth_services_test.py

```
@@ -424,49 +409,56 @@ def revoke_refresh_tokens(self, uid: str) -> None:
-    def update_user(
-        self,
-        uid: str,
-        email: Optional[str] = None,
-        disabled: bool = False,
-        custom_claims: Optional[str] = None,
-    ) -> str:
+    # Here we use type Any because `_set_user_fragile` is responsible for giving
+    # the properties strong types. This method stubs a public function in the
+    # Firebase Admin SDK, so it'd introduce even more fragility and maintenance
+    # if we try duplicate the types here.
+    def update_user(self, uid: str, **kwargs: Any) -> firebase_auth.UserRecord:
```

Ruff (0.14.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)